source/features/rgh-linkify-yolo-issues.tsx

```diff
@@ -0,0 +1,31 @@
+import * as pageDetect from 'github-url-detection';
+
+import React from 'dom-chef';
+
+import select from 'select-dom';
+
+import features from '../feature-manager';
+
+import {wrap} from '../helpers/dom-utils';
+
+import {getRghIssueUrl} from '../helpers/rgh-issue-link';
+import {isAnyRefinedGitHubRepo} from '../github-helpers';
```

Comment:
```suggestion
import React from 'dom-chef';
import select from 'select-dom';
import * as pageDetect from 'github-url-detection';

import {wrap} from '../helpers/dom-utils';
import features from '../feature-manager';
import {getRghIssueUrl} from '../helpers/rgh-issue-link';
import {isAnyRefinedGitHubRepo} from '../github-helpers';
```

Small lint just to remove the extra spaces